libomptarget/plugins/cuda/src/ompt-cupti.cpp

```diff
@@ -188,13 +190,12 @@ static const char *ompt_documentation =
 // values set by initializer
 //--------------------------
 
-static std::atomic<uint64_t> cupti_active_count(0);
 
 static bool ompt_enabled = false;
 static bool ompt_initialized = false;
 
-static libomptarget_get_target_info_t        libomptarget_get_target_info;
-
+static libomptarget_get_target_info_t libomptarget_get_target_info;
+static libomptarget_correlation_callback_t libomptarget_correlation_callback;
```

Comment:
`libomptarget_get_target_info` and `libomptarget_correlation_callback` are similar

Reply:
Only one of them is necessary. I think I renamed the libomptarget_correlation_callback to libomptarget_get_target_info and forgot that I did that.